cve_bin_tool/util.py

```diff
@@ -71,18 +72,20 @@ class VersionInfo(NamedTuple):
     end_excluding: str
 
 
-class CVEData(defaultdict):
-    def __missing__(self, key):
+class CVEData(DefaultDict[str, Union[List[CVE], Set[str]]]):
+    def __missing__(self, key: str) -> list[CVE] | set[str]:
```

Comment:
Should this one have a return type?  I'm not sure if those are needed for __ functions.

Reply:
We do already use type hints for __ functions, I think we should have them, to be consistent.